tdrs-backend/tdpservice/settings/common.py

```diff
@@ -106,10 +70,14 @@ class Common(Configuration):
 
     ALLOWED_HOSTS = ["*"]
     ROOT_URLCONF = "tdpservice.urls"
-    SECRET_KEY = os.getenv("DJANGO_SECRET_KEY", token_urlsafe(50))
+    SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
```

Comment:
The generated default here causes login failures and other issues because each time the setting was accessed it would cause a new value to be generated, breaking anything that was previously encrypted (like request sessions) since we would not be able to decrypt it with a newly generated key.